src/us/kbase/genbank/GbkParser.java

```diff
@@ -121,20 +125,10 @@ public static void parse(BufferedReader br, GbkParsingParams params, String file
                             }
                         }
                         if (qual_name != null) {
-                            line = line.substring(equal_pos + 1).trim();
-                            if (qual != null) qual.close();
-                            //System.out.println(":" + line + ":");
-                            if ((line.indexOf("/function=") != -1 || line.indexOf("/note=") != -1) && line.indexOf("/protein_id=") != -1) {
-                                //System.out.println(":" + line + ":");
-                                int chop = line.indexOf("/protein_id=");
-                                line = line.substring(0, chop);
-                            }
-                            if (!line.endsWith("\""))
-                                line += "\"";
-                            if (!line.equals("\"")) {
-                                qual = new GbkQualifier(line_num, qual_name, line);
-                                feat.qualifiers.add(qual);
-                            }
+			    line = line.substring(equal_pos+1).trim();
+			    if(qual!=null) qual.close();

```

Comment:
This section was reverted from the non-working code in the "develop" branch to working code from the "master" branch.  At some point, somebody in the develop branch had changed the spacing, so my change restored the original.